### PR TITLE
initial page for nttw5

### PR DESCRIPTION
**Pull request by dericed**

### CODE REVIEW

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 1]
```diff
@@ -31,19 +31,18 @@
 <p>The conference will be livestreaming on the MediaArea YouTube channel <a href="http://www.youtube.com/c/MediaAreaNetOfficial">available here</a>.</p>
 
 <h2>Where</h2>
-<p><a href="www.beeldengeluid.nl">The Netherlands Institute for Sound and Vision</a>, Media Parkboulevard 1, 1217 WE Hilversum, Netherlands</p>
+<p><a href="//www.beeldengeluid.nl">The Netherlands Institute for Sound and Vision</a>, Media Parkboulevard 1, 1217 WE Hilversum, Netherlands.</p>
```
Comment (retokromer) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:34]: `https://www.beeldengeluid.nl/` ?

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+  <img src="{{ asset_url }}" alt="exterior image of the Netherlands Institute for Sound and Vision" class="img-responsive center-xs" title="The Netherlands Institute for Sound and Vision">
+{% endimage %}
+<em>Photo Credit: <a href="https://commons.wikimedia.org/wiki/File:Erfgoed_Gelderland_presentatie_Jesse_de_Vos.pdf">85jesse</a></em>
+</p>
+
+<p>
+<iframe src="https://www.google.com/maps/embed?pb=!1m18!1m12!1m3!1d2443.436270359645!2d5.170862851972044!3d52.23545897966196!2m3!1f0!2f0!3f0!3m2!1i1024!2i768!4f13.1!3m3!1m2!1s0x47c66b5af17f1a73%3A0xd62b509149ca15ac!2sNetherlands%20Institute%20for%20Sound%20and%20Vision!5e0!3m2!1sen!2sus!4v1579973749555!5m2!1sen!2sus" width="600" height="450" frameborder="0" style="border:0;" allowfullscreen=""></iframe>
+</p>
+
+<p>Parts of the conference will also be accessible remotely. More info soon.</p>
```
Comment (retokromer) [inline src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig]: I personally would not say `soon`.

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig (hunk 1 above)
Comment (JeromeMartinez) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:34]: [Protocol-relative URL](https://en.wikipedia.org/wiki/URL#prurl), I use to let the user keeping the protocol used on the page even if nowadays lot of people advocate for switching to all in HTTPS. And trailing slash for root page is less used nowadays (hidden in browsers...).

Comment (retokromer) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:34]: Thank you for the information! I didn’t know that. Yet why not for the other links as well?

Comment (JeromeMartinez) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:34]: Some incoherencies with different contributors (and I also sometimes forget to do it), I need to normalize that at some point.

Comment (retokromer) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig:34]: I can open a PR once this one has been merged.

File: src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig (hunk 2 above)
Comment (JeromeMartinez) [reply src/NoTimeToWaitBundle/Resources/views/Default/nttw5.html.twig]: `soon` removed.

**Review by JeromeMartinez — APPROVED** [review]

**Comment by JeromeMartinez** [comment]
https://mediaarea.net/NoTimeToWait5 (and also https://mediaarea.net/nttw5 if you wish a tiny URL) is online.

**Pull request merged**